### DISCUSSION TITLE
Browse data on a map

### DISCUSSION THREAD

**Discussion by OzPerle** [OvertureMaps/data#11] [Discussion]
I really like this project and the idea behind making the data openly available. When are you planning to provide a simple map so the majority of users can simply browse and review the data visually?

**Comment by drewda** [comment]
To view the places layer on a slippy map, see: https://bdon.github.io/overture-tiles/places.html

Vector tiles were created by @bdon using the Tippecannoe tiling package. Processing steps listed in https://github.com/bdon/overture-tiles/blob/main/README.md

EDIT: I see this question is in the transportation category. The links above are only relevant to the places layer for now.

**Comment by msbarry** [comment]
I also created this map using planetiler with all the layers (except transportation connectors) for the planet: https://msbarry.github.io/planetiler-overture-demo/

**Reply by OzPerle** [reply]
Well done! This is great.
